src/labels/CustomLabels.labels

```diff
@@ -871,6 +871,14 @@
         <shortDescription>Setting Course Description Copy Email Subject</shortDescription>
         <value>Course Description Copy Job Completed with a status of:</value>
     </labels>
+    <labels>
```

Comment:
These still need to be reviewed by Julie but I will try to run it by her in the next day or two.

Reply:
Currently discussing with @juliesnow in Quip and hope to resolve by EOD or Monday